src/main/kotlin/uk/gov/justice/digital/hmpps/manageusersapi/service/RolesService.kt

```diff
@@ -1,29 +1,21 @@
 package uk.gov.justice.digital.hmpps.manageusersapi.service
 
-import org.springframework.http.HttpStatus
 import org.springframework.stereotype.Service
-import org.springframework.web.reactive.function.client.WebClient
-import org.springframework.web.reactive.function.client.WebClientResponseException
 import uk.gov.justice.digital.hmpps.manageusersapi.resource.Role
+import uk.gov.justice.digital.hmpps.manageusersapi.resource.RoleNameAmendment
 
 @Service
 class RolesService(
-  val authWebClient: WebClient,
+  val authService: AuthService,
 ) {
 
   @Throws(RoleNotFoundException::class)
   fun getRoleDetail(roleCode: String): Role {
```

Comment:
Outside scope of this PR, but interestingly we could make this reactive and non-blocking :-)

Reply:
Will chat to you about this.